src/locales/de.yaml

```diff
@@ -262,9 +262,16 @@ app:
       mjpegadaptive: MJPEG Adaptive
       mjpegstream: MJPEG Stream
       video: IP Camera
+    camera_rotate_options:
+      none: Keine
```

Comment:
"Drehen um: Keine" seems a bit off, maybe replace this with "Keine Drehung" to be more verbose?